### DISCUSSION TITLE
Apache convert vs Root convert from PDF to JPEG

### DISCUSSION THREAD

**Discussion by lee-ry** [ImageMagick/ImageMagick#2029] [Help]
Hi,

When i use **root** to run convert for a PDF file with unicode characters, it can convert perfectly into JPEG but when i use **apache** to run the same command to convert the unicode PDF to JPEG, the unicode characters has become symbols.

Can i know why?

**Comment by urban-warrior** [comment]
The Apache environment differs from the environment @ root.  You could, for example, be running two different versions of ImageMagick, 1 for root, 1 for Apache.  Make sure you run ImageMagick with an absolute path (e.g. /usr/local/bin/magick).  Also dump your environment variables and see how they differ in the two environments.  Your root environment, for example, might have a UTF-8 locale whereas Apache may not.
